src/index.js

```diff
@@ -5,107 +5,209 @@ import './view/css/style.css';
+let currentProjectIndex = null;
 
+// loads home
+const main = document.querySelector('#content');
 main.insertAdjacentHTML('beforeEnd', '<div class="activeInfo"></div>');
 main.insertAdjacentHTML('beforeEnd', footer);
-
 const activeInfo = document.querySelector('.activeInfo');
 activeInfo.innerHTML = home;
 
+// save to local storage
+function saveLocal() {
+  localStorage.setItem('projectList', JSON.stringify(projectList));
+}
 
-const project = (pName) => {
-  const name = pName;
-  let taskArr = [];
+// add project to projectList, save local and reload
+function addProject() {
+  const projectTitle = document.getElementById('new-project-title').value;
+  const newProject = new Project(projectTitle);
+  projectList.push(newProject);
+  saveLocal();
+  location.reload(); // eslint-disable-line no-restricted-globals
+}
 
-  const addTask = (Arr, task) => {
-  }
+function editProject(projectIndex) {
+  const newName = document.getElementById('edit-project-title').value;
+  projectList[projectIndex].title = newName;
 
-  const deleteTask = () => {
+  saveLocal();
+  renderProjects(projectList); // eslint-disable-line no-use-before-define
+}
 
-  }
+function setProjectModal(projectIndex) {
+  document.getElementById('edit-project-title').value = projectList[projectIndex].title;
+}
 
-  const setName = (newName) => {
-    this.name = newName;
-  }
+// remove project from projectList, save local and render
+function removeProject(project) {
+  projectList.splice(projectList.indexOf(project), 1);
+  alert(`${project.title} deleted`); // eslint-disable-line no-alert
 
-  const getName = () => {
-    return this.name;
-  }
+  saveLocal();
+  location.reload(); // eslint-disable-line no-restricted-globals
+}
 
-  return {
-    addTask,
-    deleteTask,
-    setName,
-    getName,
-    taskArr
-  };
+function removeTodo(project, todo) {
+  project.todos.splice(project.todos.indexOf(todo), 1);
+  alert(`${todo.title} deleted`); // eslint-disable-line no-alert
 
+  saveLocal();
+  location.reload(); // eslint-disable-line no-restricted-globals
 }
 
-const task = (tName, tDescription, tDate, tChecked) => {
-  const name = tName;
-  const description = tDescription;
-  const date = tDate;
-  const checked = tChecked;
-
-  const setName = (newName) => {
-    this.name = newName;
-  }
-
-  const getName = () => {
-    return this.name;
-  }
-
-  const setDescription = (newDescription) => {
-    this.description = newDescription;
-  }
-
-  const getDescription = () => {
-    return this.description;
-  }
-
-  const setDate = (newDate) => {
-    this.date = newDate;
-  }
-
-  const getDate = () => {
-    return this.date;
-  }
-
-  const changeChecked = () => {
-    this.checked = !this.checked;
-  }
-
-  const getChecked = () => {
-    return this.checked;
-  }
-
-  return {
-    setName,
-    setDescription,
-    setDate,
-    changeChecked,
-    getName,
-    getDescription,
-    getDate,
-    getChecked
-  };
+function editTodo(todoIndex) {
```

Comment:
You ought to use ES6 Syntax for this project:
Which means usage of arrow functions instead of the function keyword.
See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from pathwright